dashboard/app/assets/javascripts/client_state.js

```diff
@@ -0,0 +1,78 @@
+//= require jquery
+//= require jquery.cookie
+
+(function (window, $) {
+
+if (!window.dashboard) {
+  window.dashboard = {};
+}
+
+dashboard.clientState = {};
+
+/**
+ * Number of days before client state cookie expires.
+ * @type {number}
+ * @private
+ */
+dashboard.clientState.EXPIRY_DAYS = 365;
+
+dashboard.clientState.reset = function() {
```

Comment:
I added the module comment I had previously forgotten and made a note there about future things we'll add.